### DISCUSSION TITLE
俄羅斯限制惰性氣體出口，全球晶片荒恐加劇

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#646] [General]
https://technews.tw/2022/06/03/russia-restricts-inert-gas-exports/

烏克蘭是全球惰性氣體主要生產國，惰性氣體的生產關係到半導體晶片產業的發展。
如今俄羅斯入侵烏克蘭惰性氣體生產城市馬立波，恐怕將造成全球IC產業股市波動，也影像著晶片3C產品的市場價格。

**Comment by yenwei12138** [comment]
https://udn.com/news/story/6853/6286913
不只半導體，連汽車、航空、化工產業的原物料供應也大受影響。

**Comment by treeann** [comment]
惰性氣體的攻擊力很強ㄤ 許多晶片的供應兜售倒許多的影響

**Comment by ghost** [comment]
感覺因應之道還是有的 不太可能衝突爆發前沒想過

**Comment by Lai27330** [comment]
感覺中短期應該是還有惰性氣體庫存，但如果真的拖太久就麻煩了，也希望戰爭能趕快結束。

**Comment by etin2468** [comment]
又要再度陷入晶片荒了喔，東西又要變貴了哀

**Comment by stan890425** [comment]
不要在晶片荒了啦~~我都買不到PS5

**Comment by Yuhsin000** [comment]
晶片荒不知道股價是漲是跌

**Comment by sys511613** [comment]
希望戰爭能趕快停止，不然真的要什麼都買不起了QQ

**Comment by leon10456** [comment]
希望世界不要有戰爭.....

**Comment by jcpan821** [comment]
打仗真的百害無一利

**Comment by Jimmy911218** [comment]
希望屋惡戰爭趕緊結束
